### ISSUE TITLE
descriptor or coefficients

### ISSUE THREAD

**Issue by bobchennan** [lolemacs/soft-sharing#1]
In [main.py](https://github.com/lolemacs/soft-sharing/blob/master/main.py#L147) you used name "descriptor". But in your [layer implementation](https://github.com/lolemacs/soft-sharing/blob/master/models/layers.py#L23) I think it is called "coefficients".

Is "descriptor" a special term in Pytorch or you actually did apply weight decay on those coefficients?

**Comment by lolemacs** [comment]
Just changed to 'coefficients' in main.py (the two names should indeed match). Thanks for catching this.
